src/test/java/seedu/contax/logic/parser/ChainCommandParseTest.java

```diff
@@ -0,0 +1,63 @@
+package seedu.contax.logic.parser;
+
+import static seedu.contax.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.contax.logic.commands.CommandTestUtil.ADDRESS_DESC_AMY;
+import static seedu.contax.logic.commands.CommandTestUtil.EMAIL_DESC_AMY;
+import static seedu.contax.logic.commands.CommandTestUtil.NAME_DESC_AMY;
+import static seedu.contax.logic.commands.CommandTestUtil.PHONE_DESC_BOB;
+import static seedu.contax.logic.commands.CommandTestUtil.TAG_DESC_FRIEND;
+import static seedu.contax.logic.commands.CommandTestUtil.TAG_DESC_HUSBAND;
+import static seedu.contax.logic.commands.CommandTestUtil.VALID_ADDRESS_AMY;
+import static seedu.contax.logic.commands.CommandTestUtil.VALID_EMAIL_AMY;
+import static seedu.contax.logic.commands.CommandTestUtil.VALID_NAME_AMY;
+import static seedu.contax.logic.commands.CommandTestUtil.VALID_PHONE_BOB;
+import static seedu.contax.logic.commands.CommandTestUtil.VALID_TAG_FRIEND;
+import static seedu.contax.logic.commands.CommandTestUtil.VALID_TAG_HUSBAND;
+import static seedu.contax.logic.parser.CommandParserTestUtil.assertParseFailure;
+import static seedu.contax.logic.parser.CommandParserTestUtil.assertParseSuccess;
+import static seedu.contax.testutil.TypicalIndexes.INDEX_SECOND_PERSON;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.contax.commons.core.index.Index;
+import seedu.contax.logic.commands.AddCommand;
+import seedu.contax.logic.commands.ChainCommand;
+import seedu.contax.logic.commands.Command;
+import seedu.contax.logic.commands.EditCommand;
+import seedu.contax.testutil.EditPersonDescriptorBuilder;
+import seedu.contax.testutil.PersonBuilder;
+
+
+public class ChainCommandParseTest {
```

Comment:
Please check this file name.
```suggestion
public class ChainCommandParserTest {
```

Reply:
thx changed alr